microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -113,123 +129,60 @@ keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft S
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Web App Manifest][MDNWebAppManifest]
-   :::column-end:::
-   :::column span="2":::
-      Provides a JSON-based metadata file that describes key information about your web app, so that Windows 10 and other host platforms provide your PWA users with an installable, native app-like experience.  Key information includes icons, language, and URL entry point.
-   :::column-end:::
-:::row-end:::
+## The Microsoft Store
 
-To be a great PWA, your app must also meet the following requirements.
+Because PWAs are first-class citizens in the Microsoft Store, users can fully engage with them, from discovery, to installation, to execution, without ever opening the browser.
 
-:::row:::
-   :::column span="1":::
-      [Cross-browser compatibility][MDNCrossBrowserTesting]
-   :::column-end:::
-   :::column span="2":::
-      Ensure your PWA works by [testing][MicrosoftDeveloperEdgeToolsRemote] in different browsers and environments.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Responsive design][WikiResponsiveWebDesign]
-   :::column-end:::
-   :::column span="2":::
-      Employs fluid layouts and flexible images.  Responsive design includes the following elements that adapt your UX to your user's device.
-
-      *   CSS [grid][MDNCssGridLayout]
-      *   [flexbox][MDNCssFlexibleBoxLayout]
-      *   CSS [grid][MDNCssGridLayout] and [flexbox][MDNCssFlexibleBoxLayout]
-      *   [media queries][MDNMediaQueries]
-      *   [responsive images][MDNResponsiveImages]
-
-      Uses [device emulation tools][DevToolsGuideDeviceModeTestingOtherBrowsers] from your browser to locally test, or create a remote debugging session on [Windows][DevtoolsRemoteDebuggingWindows] or [Android][DevtoolsRemoteDebuggingIndex] to test directly on a target device.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Deep linking][WikiDeepLinking]
-   :::column-end:::
-   :::column span="2":::
-      Routes each page of your site to a unique URL so existing users may help you engage an even broader audience through social media sharing.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Validation and testing practices][Webhint]
-   :::column-end:::
-   :::column span="2":::
-      Uses code quality tools like the [Webhint][Webhint] linter to optimize the efficiency, robustness, safety, and accessibility of your app.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Chromium PWA Checklist][WebDevGoodPwaChecklist]
-   :::column-end:::
-   :::column span="2":::
-      Verifies your PWA against the Google baseline PWA checklist.
-   :::column-end:::
-:::row-end:::
+As the most used app on PCs, the Microsoft Store provides a trustworthy and familiar experience for your users to install your app. Additionally, it provides you with analytics.
```

Comment:
"Additionally, it provides you with analytics." feels awkward. I don’t know if there is something we can do to add to this sentence that provides more detail around the benefits, but the sentence could use more _umph_.

Reply:
consider:   "usage analytics".    Look at an analytics page at the store and give hints here about what valuable info is there.  Or maybe this set of articles already describes that and you can copy that, summarizing here.

Reply:
I've made a change which I'll push in a minute. But maybe @jgw96 can take a look at it and suggest better phrasing.